### PR TITLE
clairpostgres: fail if no config is provided (PROJQUAY-3441)

### PR DESCRIPTION
**Pull request by ricardomaraschini**
If clairpostgres component is unmanaged, clair is managed but no clair
database information has been provided by the user we now fail.

### CODE REVIEW

File: apis/quay/v1/quayregistry_types.go [hunk 1]
```diff
@@ -30,12 +30,18 @@ import (
 	quaycontext "github.com/quay/quay-operator/pkg/context"
 )
 
+// QuayVersion represents a quay version as a string. Normally this is set using semantic
+// versioning, e.g. 3.7.0.
```
Comment (dmage) [inline apis/quay/v1/quayregistry_types.go:34]: is it a build-time constant?

Comment (ricardomaraschini) [reply apis/quay/v1/quayregistry_types.go:34]: nope, the value is set at runtime (env var)

**Review by jonathankingfc — APPROVED** [review]
LGTM

**Pull request merged**